client/components/catalog/Card/Tags/index.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <v-chip
+      v-for="tag in repository.tags"
+      :key="tag.id"
+      @click:close="showDeleteConfirmation(tag)"
+      color="grey darken-1"
+      text-color="white"
+      close>
+      {{ tagName(tag) }}
+    </v-chip>
+    <add-tag v-if="exceedTagLimit" :repository="repository" />
+  </div>
+</template>
+
+<script>
+import AddTag from './AddTag';
+import EventBus from 'EventBus';
+import { mapActions } from 'vuex';
+import truncate from 'lodash/truncate';
+
+const appChannel = EventBus.channel('app');
+const TAG_LIMIT = 3;
+
+export default {
+  props: {
+    repository: { type: Object, required: true }
+  },
+  computed: {
+    exceedTagLimit: ({ repository }) => repository.tags.length < TAG_LIMIT
```

Comment:
This one is actually showing if it is under tag limit. Change cond to > or rename.